tests/test_utils.py

```diff
@@ -67,3 +67,9 @@ def test_ucblist_blocks_dict(self):
         self.assertEqual(300, len(blocks))
         self.assertEqual((0, 127, 'Basic Latin'), blocks[0])
         self.assertEqual((1048576, 1114111, 'Supplementary Private Use Area-B'), blocks[-1])
+
+    def test_smart_round(self):
+        nums = [0.41, 0.093, 0.002, 1.59, 0.0079, 0.080, 0.375, 0, 10.1, -0.061]
```

Comment:
test if rounding up also works when num < 0
e.g. `-0.067` should be rounded to `-0.07`